### PR TITLE
Feature/waydes 22

### PR DESCRIPTION
**Pull request by Tom-w2w**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,11 +42,12 @@
     "lint-staged": "^10.5.4",
     "mini-css-extract-plugin": "^0.9.0",
     "npm-run-all": "^4.1.5",
+    "postcss": "^7.0.35",
     "postcss-loader": "^3.0.0",
     "prettier": "^2.2.1",
     "rimraf": "^3.0.2",
     "serve": "^11.3.2",
-    "tailwindcss": "^1.6.2",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.1.0",
```
Comment (elwinvaneede) [inline package.json:50]: **question**: I don't understand this construction, why is this needed?

File: src/components/textarea.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+<way-textarea type="text" name="textarea" label="label" rows="10" disabled></way-textarea>
+
+```html
+<way-textarea type="text" name="textarea" label="label" rows="10" disabled></way-textarea>
+```
+
+### Max characters
+
+Use the `autofocus` attribute to set automic focus on the textarea.
```
Comment (elwinvaneede) [inline src/components/textarea.md:65]: **issue**: Incorrect content for this heading.

File: src/styleguide.md [hunk 3]
```diff
@@ -4,31 +4,260 @@ title: Styleguide - Way2Web Design System
 
 ## Styleguide
```
Comment (elwinvaneede) [inline src/styleguide.md:5]: **suggestion**: I would call this page something like "Customizing"

File: src/styleguide.md [hunk 4]
```diff
@@ -4,31 +4,260 @@ title: Styleguide - Way2Web Design System
...
+    </div>
+  </div>
+
+  <div class="mb-12">
+    <legend class="mb-2 font-bold border-b">
+      Font-sizes
+    </legend>
+    <div class="flex items-center mb-8">
+        <span class="mr-3 w-72">--way-font-size-xx-small (10px)</span>
+        <span class="text-xxSmall">The quick brown fox...</span>
```
Comment (elwinvaneede) [inline src/styleguide.md:205]: **suggestion**: I would make this the full sentence: "The quick brown fox jumps over the lazy dog". Because it contains all of the letters of the English alphabet.

File: tailwind.config.js [hunk 5]
```diff
@@ -8,6 +8,36 @@ module.exports = {
   theme: {
     extend: {
       colors: {
+        primary: '#1799de',
```
Comment (elwinvaneede) [inline tailwind.config.js:11]: **issue**: can we make this `'var(--way-color-primary)'` etc.? This way it's in sync with the defaults of the `@w2wds/core` package. Should work because we include `w2wds/core.css`.

**Review by elwinvaneede — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Tom-w2w) [reply package.json:50]: https://tailwindcss.com/docs/installation#post-css-7-compatibility-build

Comment (elwinvaneede) [reply package.json:50]: Thanks for providing the explanation, now I understand.

File: tailwind.config.js (hunk 5 above)
Comment (Tom-w2w) [reply tailwind.config.js:11]: Not sure what you mean? you mean:

`primary: var(--way-color-primary)`

Comment (Tom-w2w) [reply tailwind.config.js:11]: Dont think thats possible in de config?

Comment (Tom-w2w) [reply tailwind.config.js:11]: mmh maybe this works : primary: 'var(--way-color-primary)',

Comment (elwinvaneede) [reply tailwind.config.js:11]: No as a string, which will then be used in CSS which translated to the CSS var

File: tailwind.config.js [hunk 6]
```diff
@@ -32,10 +62,27 @@ module.exports = {
           900: '#314156',
         },
       },
+
+      fontSize: {
+        xxSmall: 'var(----way-font-size-xx-small)',
```
Comment (elwinvaneede) [inline tailwind.config.js:67]: **issue**: Should be `----` instead of `--` (and for the following lines as well of course). In the deploy preview you can also see the font sizes don't display correctly.

File: tailwind.config.js [hunk 7]
```diff
@@ -8,6 +8,36 @@ module.exports = {
   theme: {
     extend: {
       colors: {
```
Comment (elwinvaneede) [inline tailwind.config.js:10]: **issue**: Looks good. Some time ago I introduced some more colors: `tertiary`, `success` & `warning`. Would be nice if you could also add those here and in the Customizing page.

**Review by elwinvaneede — CHANGES_REQUESTED** [review]

**Review by elwinvaneede — APPROVED** [review]
